components/cards/InspectionPersonsNumberCard.vue

```diff
@@ -10,6 +10,9 @@
     >
       <template v-slot:description>
         <ul>
+          <li>
+            {{ $t('（注）検査結果の判明日を基準とする。') }}
```

Comment:
都の指示どおりのテキストなのですが、文末の「。」はここではないほうがいいかもしれないですね。
```suggestion
            {{ $t('（注）検査結果の判明日を基準とする') }}
```

Reply:
おっしゃる通りですね。下に続いている注釈は「。」がないので良いと思います。